docs/_static/api.json

```diff
@@ -0,0 +1 @@
: "A PEM encoded client certificate used for authentication. Returns SHA256 sum on GET.", "type": "string"}, "ssl_client_key": {"title": "Ssl client key", "description": "A PEM encoded private key used for authentication. Returns SHA256 sum on GET.", "type": "string"}, "ssl_validation": {"title": "Ssl validation", "description": "If True, SSL peer validation must be performed.", "type": "boolean"}, "proxy_url": {"title": "Proxy url", "description": "The proxy URL. Format: scheme://user:password@host:port", "type": "string"}, "username": {"title": "Username", "description": "The username to be used for authentication when syncing.", "type": "string"}, "password": {"title": "Password", "description": "The password to be used for authentication when syncing.", "type": "string"}, "_last_updated": {"title": " last updated", "description": "Timestamp of the most recent update of the remote.", "type": "string", "format": "date-time", "readOnly": true}, "download_concurrency": {"title": "Download concurrency", "description": "Total number of simultaneous connections.", "type": "integer", "minimum": 1}, "policy": {"title": "Policy", "description": "The policy to use when downloading content. The possible values include: 'immediate', 'on_demand', and 'cache_only'. 'immediate' is the default.", "type": "string", "enum": ["immediate", "on_demand", "streamed"], "default": "immediate"}}}, "RepositorySyncURL": {"required": ["repository"], "type": "object", "properties": {"repository": {"title": "Repository", "description": "A URI of the repository to be synchronized.", "type": "string", "format": "uri"}, "mirror": {"title": "Mirror", "description": "If ``True``, synchronization will remove all content that is not present in the remote repository. If ``False``, sync will be additive only.", "type": "boolean", "default": false}}}, "GalaxyRole": {"required": ["name", "namespace"], "type": "object", "properties": {"id": {"title": "Id", "type": "string", "readOnly": true}, "name": {"title": "Name", "type": "string", "minLength": 1}, "namespace": {"title": "Namespace", "type": "string", "minLength": 1}}}, "GalaxyRoleVersion": {"required": ["name"], "type": "object", "properties": {"name": {"title": "Name", "type": "string", "minLength": 1}, "source": {"title": "Source", "type": "string", "readOnly": true}}}, "GalaxyCollection": {"required": ["name", "namespace", "version"], "type": "object", "properties": {"name": {"title": "Name", "type": "string", "minLength": 1}, "namespace": {"title": "Namespace", "type": "string", "minLength": 1}, "version": {"title": "Version", "type": "string", "minLength": 1}, "href": {"title": "Href", "type": "string", "readOnly": true}, "versions_url": {"title": "Versions url", "type": "string", "readOnly": true}}}}, "tags": [{"name": "content", "x-displayName": "Content"}, {"name": "distributions", "x-displayName": "Distributions"}, {"name": "remotes", "x-displayName": "Remotes"}, {"name": "pulp_ansible", "x-displayName": "Pulp_Ansible"}]}
```

Comment:
Is this file supposed to be checked in to version control?

Reply:
Unfortunately yes. @dkliban and I have thought lots and lots about a better way but we can't come up with one. ReDoc technology wants to actively fetch this via javascript so we need to have RTD serve up the static asset in this way.